path/package.yaml

```
@@ -0,0 +1,22 @@
+_config/lib: !include "../config/hpack-common.yaml"
+
+name:       luna-path
+version:    "0.1"
+author:     Jean-Christophe Mincke <jeanchristophe.mincke@gmail.com>
+maintainer:
+    - Jean-Christophe Mincke <jeanchristophe.mincke@gmail.com>
```

```suggestion
    - Luna Team <contact@luna-lang.org>
    - Jean-Christophe Mincke <jeanchristophe.mincke@gmail.com>
```

We just need to make it clear that you don't bear the sole responsibility for maintaining this!